client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -0,0 +1,208 @@
+		const header = (
+			<div>
+				<div className="transfer-domain-step__section-heading">
+					<span className="transfer-domain-step__section-heading-number">{ position }</span>
+					{ heading }
+				</div>
+				<div>
+					<small>{ message }</small>
+				</div>
+			</div>
+		);
+
+		const cardClasses = classNames( {
+			'hide-secondary': ! button,
+		} );
+
+		return (
+			<FoldableCard
+				className={ cardClasses }
+				header={ header }
+				summary={ button }
+				expandedSummary={ button }
+			>
+				{ explanation }
+			</FoldableCard>
+		);
+	}
+
+	getStatusMessage() {
+		const { translate } = this.props;
+		const { unlocked } = this.state;
+
+		const heading = unlocked
+			? translate( 'Domain is unlocked.' )
+			: translate( 'Unlock the domain.' );
+		const message = unlocked
+			? translate( 'Your domain is unlocked at your current registrar.' )
+			: translate(
+					"You'll need to unlock the domain at your current registrar before we can move it."
+				);
+		const explanation = translate(
+			'Your current domain provider has locked the domain to prevent unauthorized transfers.'
+		);
+
+		const button = unlocked ? (
+			<div className="transfer-domain-step__unlocked">
+				<Gridicon icon="checkmark" size={ 12 } />
+				<span>{ translate( 'Unlocked' ) }</span>
+			</div>
+		) : (
+			<Button disabled={ this.state.loading } onClick={ this.refreshStatus }>
+				{ translate( 'Check Again' ) }
+			</Button>
+		);
+
+		return this.getSection( heading, message, explanation, 1, button );
+	}
+
+	getPrivacyMessage() {
+		const { translate } = this.props;
+		const { email, privacy } = this.state;
+
+		const heading = privacy
+			? translate( 'Privacy protection is disabled.' )
+			: translate( 'Disable privacy protection.' );
+		const message = privacy
+			? translate(
+					"We'll send an email to {{strong}}%(email)s{{/strong}} to start the transfer process. Don't recognize " +
+						"that address? Then you have privacy protection, and you'll need to turn it off before we start.",
+					{
+						args: { email },
+						components: { strong: <strong /> },
+					}
+				)
+			: translate(
+					"We'll send an email to {{strong}}%(email)s{{/strong}} to start the transfer process. After the transfer" +
+						'is complete you can enable privacy to hide your registration information again.',
+					{
+						args: { email },
+						components: { strong: <strong /> },
+					}
+				);
+		const explanation = translate(
+			"It's important that we be able to reach you, because the transfer involves few emails. Privacy protection is great, " +
+				'but means that your contact information is hidden. To continue with the transfer, turn privacy protection off for now ' +
+				'- you can re-enable it once the transfer is done.'
+		);
+
+		return this.getSection( heading, message, explanation, 2 );
+	}
+
+	getEppMessage() {
+		const { translate } = this.props;
+
+		const heading = translate( 'Get a domain authorization code.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Get domain authorization code.' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).